pam_eye.c

```diff
@@ -7,45 +8,87 @@
 #include <security/pam_modules.h>
 
 #define USERAGENT "pam_eye/1.0"
-#define TIMEOUT   10L
+#define DEFAULT_TIMEOUT_MS 200
```

Comment:
Keep in mind, that PAM is used to authenticate via sudo/su as well, so server fault can lead to major slowdown of authentication system. 200ms is pretty big number in the world of networking, it's greater than RTT between continents.